test/helpers/react_components/mentoring/inbox_test.rb

```diff
@@ -2,9 +2,9 @@
 
 class MentoringInboxTest < ReactComponentTestCase
   test "mentoring inbox rendered correctly" do
-    component = ReactComponents::Mentoring::Inbox.new(
+    component = ReactComponents::Mentoring::Inbox.new({
       criteria: "Ruby"
-    )
+    })
```

Comment:
I've left this to be using `{}`. I could change this, but I'd need to call `**@params` in a HAML template.